_posts/2021-02-03-cockpit-237.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Cockpit 237
+author: marusak
+date: 2021-02-03
+tags: cockpit linux
+slug: cockpit-237
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 237.
+
+## Updates: Restyling as preparation for upcoming new fatures
+
+Updates page went through some design polishing as preparation for new upcoming features as support
+of [tracer](https://tracer.readthedocs.io/en/latest/) and [kpatch](https://github.com/dynup/kpatch).
```

Comment:
Let's please not announce vaporware.  I'd stop after "for upcoming features.". If you do want to keep it, "features, like support for tracer and kpatch".